package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
if you want, you can combine this with your other one and just have one package for the whole project